modules/veteran_verification/app/services/bgs/mock_disability_rating_service.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'savon'
+require 'nokogiri'
+
+module BGS
+  class MockDisabilityRatingService
+    def get_rating(current_user)
+      options = {
+        wsdl: "#{Settings.bgs.url}/RatingServiceBean/RatingService/ratingRecord.wsdl",
```

Comment:
Same comment... if BGS is used by multiple modules, does this setting need to be local to vet_verification?